docs/getting-started/quickstart.md

```diff
@@ -84,7 +84,7 @@ infra login <INFRA_SERVER_HOSTNAME> --skip-tls-verify
 Generate an access key:
```

Comment:
```suggestion
Generate an access key for the Kubernetes cluster connector:
```

Reply:
https://github.com/infrahq/infra/blame/main/docs/install/install-on-kubernetes.md#L126

This is the place that makes me confused.

Reply:
```
~/D/t/infra (main …1) infra keys add --help
Create an access key for a user.

Usage:
  infra keys add USER [flags]
```

Reply:
Since it's late at night in the time zone here, I'm going to go to bed first. I saw this project before I went to bed and it was great.

Reply:
That `infra keys add` usage help is confusing, it is oriented towards adding an access key for a user, but adding an access key for a connector is a very common case. I created #2004 to address this.

Thanks for checking out the project and the positive feedback!